build.gradle

```diff
@@ -1,74 +1,70 @@
 plugins {
-	id 'fabric-loom' version '0.4-SNAPSHOT'
+	id 'fabric-loom' version '0.10-SNAPSHOT'
+	id 'maven-publish'
+	id "com.github.johnrengelman.shadow" version "7.0.0"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+apply plugin: 'java'
+apply plugin: 'fabric-loom'
+apply plugin: 'maven-publish'
+apply plugin: "com.github.johnrengelman.shadow"
```

Comment:
Why are you using the legacy method of applying plugins?

Additionally, the plugins are automatically applied when added in the `plugins` DSL.